src/Request.php

```diff
@@ -1076,10 +1076,11 @@ private static function limitTelegramRequests($action, array $data = [])
 
                     $requests = DB::getTelegramRequestCount($chat_id, $inline_message_id);
 
-                    if ($requests['LIMIT_PER_SEC'] == 0     // No more than one message per second inside a particular chat
```

Comment:
Intentional, `LIMIT_PER_SEC_ALL` should also apply to groups, it didn't before.

At some point the limiter will have to be redone, there is more limits than they put in the documentation (editing has different limits than sending).